tool/tsh/proxy.go

```diff
@@ -661,6 +661,8 @@ func dbProtocolToText(protocol string) string {
 		return "Redis"
 	case defaults.ProtocolSQLServer:
 		return "SQL Server"
+	case defaults.ProtocolCassandra:
+		return "Cassandra"
```

Comment:
hmm... my bad i didn't know such function exist. I think we should merge these two
https://github.com/gravitational/teleport/blob/293712f2459f40787f3f2020586712e1fd57a408/lib/defaults/defaults.go#L501-L506